<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="earth_rover_microcontroller_bridge" pkg="earth_rover_microcontroller_bridge" type="earth_rover_microcontroller_bridge_node" required="true" output="screen">
        <param name="serial_port" type="string" value="/dev/ttyUSB0"/>
        <param name="serial_baud" type="int" value="115200"/>
    </node>
</launch>
